src/main/resources/com/linecorp/armeria/server/docs/assets/css/armeria.css

```diff
@@ -148,4 +157,15 @@ h2 {
 
 .debug-textarea {
   width: 100%;
+  max-width: 100%;
```

Comment:
Did you mean to replace width with min-width? Don't think there's any meaning to having both width and max-width.